books/tests/test_forms.py

```diff
@@ -1,39 +1,35 @@
 from books import forms
 
+
 class TestPostBookForm:
     def test_form_valid(self):
         form = forms.PostBookForm(data={})
         assert form.is_valid() is False
 
-        form = forms.PostBookForm(data={
-            'title':'Doughnot Economics'
-            })
+        form = forms.PostBookForm(data={"title": "Doughnot Economics"})
         assert form.is_valid() is False
 
-        form = forms.PostBookForm(data={
-            'title':'Lorem ipsum dolor sit amet \
+        form = forms.PostBookForm(
+            data={
+                "title": "Lorem ipsum dolor sit amet \
             consectetur adipisicing elit. \
             Asperiores quia corrupti laborum \
             veritatis eum. Maiores numquam est \
             repellat adipisci porro a fuga, itaque \
             de animi, rerum officia minima hic \
-            necessitatibus?', 'author':'Kate Raworth'})
+            necessitatibus?",
```

Comment:
Not important in this case, but good to know:

When you have a large string you can do the following:
```
data = {
    "title": (
        "Lorem ipsum dolor sit amet "
        "consectetur adipisicing elit. "
        "consectetur adipisicing elit. "
        "Asperiores quia corrupti laborum "
        "Asperiores quia corrupti laborum  "
        "veritatis eum. Maiores numquam est "
        "veritatis eum. Maiores numquam est "
        "repellat adipisci porro a fuga, itaque "
        "repellat adipisci porro a fuga, itaque "
        "de animi, rerum officia minima hic "
        "de animi, rerum officia minima hic "
        "necessitatibus?"
    ),
    "author": "Kate Raworth",
}
```
brackets not necessary, but format better.

The difference is in your version the string will contain the exact whitespace so you would have:

**"Lorem ipsum dolor sit amet&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;consectetur adipisicing elit...."**

Because the space from the beginning of the line applies!